components/inputs/input-checkbox.js

```diff
@@ -107,6 +107,15 @@ class InputCheckbox extends RtlMixin(LitElement) {
 		if (elem) elem.focus();
 	}
 
+	simulateClick() {
+		this.checked = !this.checked;
+		this.indeterminate = false;
+		this.dispatchEvent(new CustomEvent(
+			'change',
+			{bubbles: true, composed: false}
+		));
+	}
+
 	_handleChange(e) {
```

Comment:
`_handleChange` could just call into `simulateClick` as the code is the same... but I felt weird calling it from there.